### ISSUE TITLE
now() is not an available evaluation parameter

### ISSUE THREAD

**Issue by c-bpovea** [fivetran/great_expectations#3657]
Version 0.13.40 adds test for datetime evaluation parameters, but the use case when we need just the current date was not considered. 

For example, if we test with this parameter: `"$PARAMETER": "now() - timedelta(weeks=52*1000)"` as [here](https://github.com/great-expectations/great_expectations/blob/86eea6da29e834c7cbaaeb6321c4c7d3cc4adc3d/tests/test_definitions/column_map_expectations/expect_column_values_to_be_between__evaluation_parameters.json#L133) we will have a successful validation.

But, if we try to use something like this: `"$PARAMETER": "now()"` we will have the following error:
```
           except AttributeError:
                logger.warning("Unable to get store for store-type valuation parameter.")
>               raise EvaluationParameterError("No value found for $PARAMETER " + str(L[0]))
E               great_expectations.exceptions.exceptions.EvaluationParameterError: No value found for $PARAMETER ('now', 0, False)

.../great_expectations/core/evaluation_parameters.py:359: EvaluationParameterError
```

I consider that a single `now()` must be part of the acceptance datetime evaluation parameters.

**Environment:**
 - Operating System: MacOS
 - Great Expectations Version: 0.13.41

**Comment by talagluck** [comment]
Thanks for opening this issue, @c-bpovea! I agree that this doesn't sound like it's working correctly. We will review and be in touch.

**Comment by austiezr** [comment]
Hey @c-bpovea! It looks like this should be resolved by #3719; if you feel this isn't true, feel free to re-open and address.
